src/Router.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Scene, Router } from 'react-native-router-flux';
```

Comment:
Is this a similar Router to what he have been using or is this is Navigator?

Reply:
It's different but not too difficult at all, I run you through it during work time today.